transformers4rec/config/transformer.py

```diff
@@ -288,3 +288,105 @@ def build(
             vocab_size=1,
             **kwargs
         )
+
+
+@transformer_registry.register("bert")
+class BertConfig(T4RecConfig, transformers.BertConfig):
+    @classmethod
+    def build(
+        cls,
+        d_model,
+        n_head,
+        n_layer,
+        total_seq_length,
+        hidden_act="gelu",
+        initializer_range=0.01,
+        layer_norm_eps=0.03,
+        dropout=0.3,
+        pad_token=0,
+        log_attention_weights=True,
```

Comment:
Shouldn't `log_attention_weights` default be False?
I think it should be enabled only for debugging purposes